pkg/util/utils.go

```diff
@@ -79,3 +91,73 @@ func CommaStringToStringMap(s string) (map[string]string, error) {
 
 	return kvMap, nil
 }
+
+// ValidateIpFormat will check whether the string is in IP address format(#.#.#.#).
+func ValidateIpFormat(ip string) bool {
```

Comment:
Instead of using this, you can use the govalidator.IsIPv4() https://github.com/asaskevich/govalidator/blob/master/validator.go#L616